kong/api/init.lua

```diff
@@ -53,6 +53,8 @@ local function on_error(self)
     else
       return responses.send_HTTP_BAD_REQUEST(err.tbl or err.message)
     end
+  else
+    return responses.send_HTTP_BAD_REQUEST(tostring(err))
```

Comment:
since we're supposed to do "early exits" as much as possible according to our style guide, this should be done without the `else`.